pkg/scheduler/metrics/source/metrics_client_prometheus_adapt.go

```diff
@@ -36,31 +36,40 @@ const (
 	CustomNodeMemUsageAvg = "node_memory_usage_avg"
 )
 
-type CustomMetricsClient struct {
-	config *rest.Config
+type KMetricsClient struct {
```

Comment:
`KMetricsClient` is not a good name. What's the meaning of it?

Reply:
It means k8s metrics api client.